Mathlib/Algebra/Order/Monoid/Unbundled/IsUnit.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2025 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+
+import Mathlib.Algebra.Group.Units.Defs
+import Mathlib.Algebra.Order.Monoid.Canonical.Defs
```

Comment:
Does the following not fit in this file?
```
theorem IsUnit.mulLECancellable
    {M : Type*} [Monoid M] [LE M] [MulLeftMono M]
    {a : M} (ha : IsUnit a) : MulLECancellable a := fun x c h => by
  obtain ⟨u, rfl⟩ := ha
  simpa using mul_le_mul_left' h ↑u⁻¹
```

Reply:
done